src/components/RichLayout/style.css

```diff
@@ -0,0 +1,7 @@
+.richLayout {
```

Comment:
should use kebab-case formatting like: `.rich-layout {`

post-css takes care of making it available as richLayout to other modules.